nupic/research/frameworks/dendrites/functional/apply_dendrites.py

```diff
@@ -25,10 +25,22 @@
 a set of dendritic segments.
 """
 
+from collections import namedtuple
+
 import torch
 
+dendrite_output = namedtuple("dendrite_output", ["values", "indices"])
+dendrite_output.__doc__ = """
+A named tuple for outputs modified by `apply_dendrites`_.
+
+:attr values: output tensor after being modulated by dendrite activations
+:attr indices: the indices of the winning segments used to modulate the output tensor
+
+.. _apply_dendrites: nupic.research.frameworks.dendrites.functional.apply_dendrites
+"""
+
 
-def dendritic_bias_1d(y, dendrite_activations):
+def dendritic_bias(y, dendrite_activations):
```

Comment:
The "1d" was added here (and below) to follow the formatting of `torch.nn.functional`, which has the following for convolution operations:

1D -> `torch.nn.functional.conv1d`,
2D -> `torch.nn.functional.conv2d`.

What are your reasons for name change?

Reply:
Fair point. My motivation was to keep consistency with the `SparseWeights` and `SparseWeights2d` classes. But technically there should be a `SparseWeights1d` if we're committed to being as consistent with PyTorch as possible. I re-added the `1d` so that this is the case for the dendrites module.